test/js/PiskelApiTest.js

```diff
@@ -0,0 +1,228 @@
+describe('PiskelApi test suite', function() {
+  var piskelApi;
+  beforeEach(function() {
+    piskelApi = new PiskelApi();
+  });
+
+  afterEach(function() {
+    piskelApi = null;
+  })
+
+  describe('attachToPiskel', function() {
+    var iframe;
+    beforeEach(function() {
+      spyOn(window, 'addEventListener');
+      iframe = createFakeIframe();
+      piskelApi.attachToPiskel(iframe);
+    })
```

Comment:
Another missing semicolon - looks like there are several in this file.  I assume piskel's linting isn't checking for this?